packages/webdriver/src/index.js

```diff
@@ -88,8 +88,8 @@ export default class WebDriver {
         logger.setLevel('webdriver', options.logLevel)
 
         options.capabilities = options.capabilities || {}
-        options.isW3C = options.isW3C || true
-        const prototype = Object.assign(getPrototype(options.isW3C), userPrototype)
+        options.isW3C = options.isW3C === false ? false : true
+        const prototype = Object.assign(getPrototype({ isW3C: options.isW3C }), userPrototype)
```

Comment:
I think it also makes sense to add the other flags in here too , like `isChrome` , `isSauce` etc, no?

Reply:
yeah, `getPrototype` accepts `{ isW3C, isChrome, isMobile, isSauce }`. 
Agree that it makes sense to pass all of them

Reply:
@christian-bromann  updated